indico/modules/events/templates/emails/move_request_closure.txt

```diff
@@ -0,0 +1,31 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    Event move request update
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    {%- filter dedent -%}
+        {{ ('The following events have been moved:' if accept else 'Move requests of the following events have been rejected:' ) }}
+
+        {% filter underline %}Events{% endfilter %}
+        {%- for evt in events %}
+            {{ evt.title }}
+            - URL: {{ evt.external_url }}{% if accept %}, New event path: {{ evt.category.chain_titles|join(' » ') }}{% endif %}
```

Comment:
I'd just show the titles here, and maybe the URL behind it in parentheses. Having one line per event is going to be much more readable in a plaintext email.